code/game/objects/items.dm

```diff
@@ -279,7 +279,12 @@
 // for items that can be placed in multiple slots
 // note this isn't called during the initial dressing of a player
 /obj/item/proc/equipped(mob/user, slot)
+	SHOULD_CALL_PARENT(TRUE) // no exceptions
 	SEND_SIGNAL(src, COMSIG_ITEM_EQUIPPED, user, slot)
+	if(flags_equip_slot & slotdefine2slotbit(slot)) // flags_equip_slot is a bitfield
+		SEND_SIGNAL(src, COMSIG_ITEM_EQUIPPED_TO_SLOT, user)
+	else
+		SEND_SIGNAL(src, COMSIG_ITEM_EQUIPPED_NOT_IN_SLOT, user, slot)
```

Comment:
Do a single signal and make it pass if equipped or not as a variable.

Reply:
That would completely defeat the purpose of these signals

Reply:
No, you can still register to the signal and check for the var.

Reply:
That's what you do with the current signal so you're asking me to just make a duplicate signal

Reply:
normally I wouldn't be too much a fan of that kind of thing but there's nothing inherently wrong with it and in that case in particular that check is done in a lot of component code. In this context this is a prety nice pair of signals.